scripts/lib/e2e-auth.sh

```
@@ -0,0 +1,128 @@
+# e2e-auth.sh — Agent-scoped credential preparation for the E2E harness.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)